libgalois/include/katana/MemorySupervisor.h

```diff
@@ -78,19 +85,20 @@ class KATANA_EXPORT MemorySupervisor {
 
   struct ManagerInfo {
     std::unique_ptr<Manager> manager_;
-    count_t active{};
     count_t standby{};
   };
   std::unordered_map<std::string, ManagerInfo> managers_;
 
-  count_t Used() { return active_ + standby_; }
-  count_t Available() { return physical_ - Used(); }
+  count_t Available() {
+    auto rss = static_cast<katana::count_t>(
+        katana::ProgressTracer::ParseProcSelfRssBytes());
```

Comment:
Does `rss` include memory that is swapped out or is it just what's paged in at the moment?

Reply:
It is only resident.  I have a TODO in the code to worry about swap, but I think that is a second-order effect.